src/queries/multi-match-query.js

```diff
@@ -1,17 +1,20 @@
+import {assign} from 'lodash';
+
 /**
  * Construct a Multi Match query.
  *
  * @param  {Array}  fields  The field names to query over.
  * @param  {String} query   The query string.
  * @param  {String} [type='best_fields']  The type of multi_match query.
+ * @param {Object} options see docs for possible options https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-multi-match-query.html
  * @return {Object}         Multi Match query.
  */
-export default function multiMatchQuery(fields, query, type='best_fields') {
+export default function multiMatchQuery(fields, query, type='best_fields', options={}) {
```

Comment:
What do you think about removing the `type='best_fields` parameter and leaving it to the user to define the type as part of `options`? Trying to scale down the number of arguments here.